canonical-web-and-design/kubeflow-news.com
# 1422 feedback amends

## Done

Based on feedback:
- Updated index and archive pages to show full posts, with pagination and sticky newsletter signup form
- Removed "Latest" and "Archives" from navigation
- Removed newsletter signup banner

## QA
- Check out this feature branch
- Run `./run`
- Visit http://0.0.0.0:8035
- See that the pages match the [design](https://user-images.githubusercontent.com/36884067/61204401-026e1080-a6e5-11e9-8826-b3c24ced7a49.jpg)

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1422
